.stylish-haskell.yaml

```diff
@@ -0,0 +1,212 @@
+# stylish-haskell configuration file
```

Comment:
This one should probably go away, but it _might_ be a good idea to have a standard way of formatting all the oscoin-related projects.

Reply:
Yeah I think it can stay